src/components/type-group/README.md

```diff
@@ -0,0 +1,43 @@
+# Type Group
+
+A standard Type Group input.
+
+## Usage
+
+The `<TypeGroup>` component should be used along with `<TypeOption>`. However, `<TypeOption>` can be used independently.
+
+```jsx
+import { TypeGroup, TypeOption, UserIcon, EntityIcon, CheckmarkIcon } from '@nomios/web-uikit';
+
+<TypeGroup name="example" onSelect={ handleOnSelect }>
+    <TypeOption id="1" label="Person" badge={ CheckmarkIcon }>
+        <UserIcon />
+    </TypeOption>
+    <TypeOption id="2" label="Organization" badge={ CheckmarkIcon }>
+        <EntityIcon />
+    </TypeOption>
+</TypeGroup>
+```
+
+## TypeGroup props
+
+| name | type | default | description |
+| ---- | ---- | ------- | ----------- |
+| name | string | *required* | A descriptor to relate every TypeOption in the same group |
+| onSelect | function | *required* | A function to be called whenever a TypeOption changes |
+| children | node | *required* | The content to render, usually an array of <TypeOption> elements |
+| className | string | | A classname to override styles |
```

Comment:
I would remove this from the document and `spread` all the props into the root element, and include this in the README.

> Any other properties supplied will be spread to the root element.

Several components of the UIKit already do that, e.g.: Icon.

Reply:
I would also do the spreading on `TypeOption`.